### PR TITLE
add security headers

### PR DESCRIPTION
**Pull request by pjfanning**
I've tried these out on https://pekko.staged.apache.org

these changes get us an A grade on https://securityheaders.com

eg https://securityheaders.com/?q=https%3A%2F%2Fpekko.staged.apache.org%2F&hide=on&followRedirects=on

### CODE REVIEW

**Review by gmethvin — APPROVED** [review]
lgtm

**Review by Roiocam — APPROVED** [review]
LGTM

**Review by mdedetrich — APPROVED** [review]
lgtm

File: content/.htaccess [hunk 1]
```diff
@@ -55,3 +55,11 @@ RewriteRule ^japi/([^/]+)/1.0.2/(.*)$ https://nightlies.apache.org/pekko/docs/$1
 RewriteRule ^docs/([^/]+)/1.0/(.*)$ https://nightlies.apache.org/pekko/docs/$1/1.0/docs/$2 [P]
 RewriteRule ^api/([^/]+)/1.0/(.*)$ https://nightlies.apache.org/pekko/docs/$1/1.0/api/$2 [P]
 RewriteRule ^japi/([^/]+)/1.0/(.*)$ https://nightlies.apache.org/pekko/docs/$1/1.0/japi/$2 [P]
+
+# Security Headers
+Header set Strict-Transport-Security "max-age=31536000"
+Header set Content-Security-Policy "default-src 'self' https://api.github.com/ https://pekko.apache.org/ ; style-src 'self' https://pekko.apache.org/ 'unsafe-inline' ; script-src 'self' https://pekko.apache.org/ 'unsafe-inline' ; frame-src 'self' ;"
```
Comment (raboof) [inline content/.htaccess:61]: to we really want to allow calls to api.github.com? I filed https://github.com/apache/incubator-pekko-sbt-paradox/issues/110 for this, perhaps we should at least link to that from here?

**Pull request merged**